spec/components/status_tag/view_preview.rb

```diff
@@ -3,8 +3,12 @@
 module StatusTag
   class ViewPreview < ViewComponent::Preview
     Trainee.states.keys.each do |state|
```

Comment:
Should we limit this so we are just adding differentiating between EYTS and QTS for the awarded/recommended states? Might be a bit confusing otherwise.

Reply:
https://rtt-review-pr-933.herokuapp.com/view_components/status_tag/view

To navigate here and double check nothing was misaligned.

Deferirng to :balance_scale:

Reply:
Yeah, not sure if this is intended, eg, we don't have a state of EYTS_draft 
https://rtt-review-pr-933.herokuapp.com/view_components/status_tag/view/EYTS_draft

Reply:
i was just being lazy - will change!

Reply:
@FriedSock done!